examples/local-access/README.md

```diff
@@ -0,0 +1,12 @@
+## Use Cluster from Local Environment Quickstart
+
+In this project, you'll learn how to create and interact with a Dask cluster from your local environment using the Saturn Cloud service. This allows you to skip interacting with the Saturn Cloud UI almost entirely, if you want to.
```

Comment:
I think it would make sense in the readme to instruct users to do all the things that would need to happen in saturn, then its clear that all the notebooks should be run from their local environment